src/lib/rest/rest.cpp

```diff
@@ -413,17 +414,26 @@ static int outFormatCheck(ConnectionInfo* ciP)
 
 /* ****************************************************************************
 *
-* servicePathCheck - 
+* servicePathCheck - check vector of service paths
```

Comment:
Do we want to have both functions (the one here and the other in string.cpp) with the same name? Of course they are different, as they have different paramters, but having the same name for both is a bit confusing, isn't it?